lib/src/sdk_env.dart

```diff
@@ -8,43 +8,46 @@ import 'dart:io';
 
 import 'package:cli_util/cli_util.dart' as cli;
 import 'package:path/path.dart' as p;
+import 'package:pub_semver/pub_semver.dart';
 
 import 'analysis_options.dart';
 import 'logging.dart';
 import 'utils.dart';
 
+final _sdkRegexp =
```

Comment:
can we get await with a simpler if (version.startsWith('Dart VM version:)) {
   // get first string after colon
}?
the date and the rest is not really important. Maybe store the original value and extract only the version?